packages/node/src/login/AggregateLoginHandler.ts

```diff
@@ -32,14 +32,18 @@ import {
   ILoginHandler,
   ILoginOptions,
   AggregateHandler,
+  ISessionInfo,
 } from "@inrupt/solid-client-authn-core";
 
 /**
  * @hidden
  */
 @injectable()
 export default class AggregateLoginHandler
-  extends AggregateHandler<[ILoginOptions], void>
+  extends AggregateHandler<
+    [ILoginOptions],
+    (ISessionInfo & { fetch: typeof fetch }) | undefined
```

Comment:
Isn't this just the type `LoginResult`...?

Reply:
Correct, I named the type as a shorthand when I realized I was using it in a lot of places, but I didn't look for it everywhere it seems. Fixed in 6ee8aac0c890e90d7e3af0fdf265de1327bb0a53